subprojects/junit-jupiter/src/test/java/org/mockitousage/StrictnessTest.java

```diff
@@ -118,6 +118,26 @@ void by_default_configures_strict_stubs_in_runner() {
         assertThat(result.getThrowable().get()).isInstanceOf(UnnecessaryStubbingException.class);
     }
 
+    @MockitoSettings(strictness = Strictness.WARN)
+    static class BaseWarnStubs {}
+
+    static class InheritedWarnStubs extends BaseWarnStubs {
+        @Mock
+        private Function<Integer, String> rootMock;
+
+        @Test
+        void should_throw_an_exception_on_strict_stubs_configured_by_default() {
```

Comment:
Nit: test name should be updated to state that it properly inherits from the MockitoSettings on the superclass.

Reply:
Done